test/test.js

```diff
@@ -1617,9 +1679,9 @@ describe('koaspec', function () {
         expect(actual).to.eql(false);
       });
 
-      it('parses a "false" string value.', function* () {
-        const actual = utils.parseBoolean('false');
-        expect(actual).to.eql(false);
+      it('parses a "true" string value.', function* () {
+        const actual = utils.parseBoolean('true');
+        expect(actual).to.eql(true);
```

Comment:
Yes, duplicate test case :
- https://github.com/luxe-eng/koa-spec/blob/master/test/test.js#L1615
- https://github.com/luxe-eng/koa-spec/blob/master/test/test.js#L1620

I assumed the original intention was to also test `true` :)